src/lib/site-import/status.js

```diff
@@ -162,6 +162,7 @@ export async function importSqlCheckStatus( { app, env }: ImportSqlCheckStatusIn
 				debug( { createdAt, dbOperationInProgress, importInProgress, importStepProgress } );
 
 				if (
+					importProgress &&
 					importStepProgress.started_at &&
```

Comment:
```suggestion
					importStepProgress?.started_at &&
```
I suppose we could use the null chaining operator here? It seems we don't make much use of this elsewhere.